src/main/java/com/mapzen/valhalla/Instruction.kt

```diff
@@ -75,22 +74,22 @@ public open class Instruction {
     protected constructor() {
     }
 
-    public fun getIntegerInstruction(): Int {
+    fun getIntegerInstruction(): Int {
        return turnInstruction;
     }
 
-    public fun getHumanTurnInstruction(): String? {
+    fun getHumanTurnInstruction(): String? {
         return json?.getString("instruction");
     }
 
-    public fun skip(): Boolean {
+    fun skip(): Boolean {
```

Comment:
No longer used in library or app. I believe this is legacy code from OSRM. Safe to remove?